Api/Validators/Visit/CreateVisitRequestValidator.cs

```diff
@@ -0,0 +1,54 @@
+﻿using FluentValidation;
+using Microsoft.AspNetCore.Identity;
+using Reservant.Api.Models;
+using Reservant.Api.Models.Dtos.Visit;
+
+namespace Reservant.Api.Validators.Visit;
+
+/// <summary>
+/// Validator for CreateVisitRequest
+/// </summary>
+public class CreateVisitRequestValidator : AbstractValidator<CreateVisitRequest>
+{
+    /// <inheritdoc />
+    public CreateVisitRequestValidator(UserManager<User> userManager)
+    {
+        RuleFor(v => v.Date)
+            .Must(date => date >= DateOnly.FromDateTime(DateTime.Now))
+            .WithMessage("The date must be today or in the future.");
```

Comment:
Dla takich zasad trzeba tworzyć customowe walidatory (w klasie CustomValidators) dla ponownego użycia w przyszłości oraz żeby wszystkie one były w jednym miejscu.

Oprócz tego trzeba też podać `.WithErrorCode()` (error code musi myć w klasie ErrorCodes), bo tego będą używać we froncie

Reply:
7f37ce2d5e319b1496f1c90b756136f657bec2e7